diff --git a/promptsource/seqio_tasks/tasks.py b/promptsource/seqio_tasks/tasks.py
index 6b39719..6004aa7 100644
--- a/promptsource/seqio_tasks/tasks.py
+++ b/promptsource/seqio_tasks/tasks.py
@@ -12,6 +12,7 @@ from t5.evaluation import metrics as mt
 
 import promptsource.templates
 from promptsource.seqio_tasks import utils
+from promptsource.utils import get_dataset_data_dir
 
 
 GET_METRICS = {
@@ -59,7 +60,9 @@ def maybe_get_class_id_postprocessor(template):
 def get_tf_dataset(split, shuffle_files, seed, dataset_name, subset_name, template, split_mapping):
     # HF datasets does not support file-level shuffling
     del shuffle_files, seed
-    dataset = datasets.load_dataset(dataset_name, subset_name)
+
+    data_dir = get_dataset_data_dir(dataset_name, subset_name)
+    dataset = datasets.load_dataset(dataset_name, subset_name, data_dir=data_dir)
     dataset = dataset[split_mapping[split]]
     dataset = utils.apply_template(dataset, template)
     return utils.hf_dataset_to_tf_dataset(dataset)
diff --git a/promptsource/utils.py b/promptsource/utils.py
index 5817b35..3613cd1 100644
--- a/promptsource/utils.py
+++ b/promptsource/utils.py
@@ -49,7 +49,23 @@ def get_dataset(path, conf=None):
         builder_instance.download_and_prepare()
         return builder_instance.as_dataset()
     else:
-        return datasets.load_dataset(path, conf)
+        data_dir = get_dataset_data_dir(path, conf)
+        return datasets.load_dataset(path, conf, data_dir=data_dir)
+
+MANUALLY_DOWNLOADED_DATASETS = {
+    "story_cloze": {
+        "2016": "/home/thomas/custom_datasets/story_cloze/2016",
+    }
+}
+def get_dataset_data_dir(dataset_name, subset_name):
+    if dataset_name in MANUALLY_DOWNLOADED_DATASETS:
+        if subset_name in MANUALLY_DOWNLOADED_DATASETS:
+            data_dir = MANUALLY_DOWNLOADED_DATASETS[dataset_name][subset_name]
+        else:
+            raise ValueError(f"Unknown subset {subset_name} for dataset {dataset_name}")
+    else:
+        data_dir = None
+    return data_dir
 
 
 def get_dataset_confs(path):
